.yamllint

```diff
@@ -41,5 +41,4 @@ rules:
     level: error
   truthy:
     level: error
-    ignore: |
-      *linter.yml
+    check-keys: false
```

Comment:
So this is about if a key in a dictionary == True? Do we have that anywhere in our code?

Reply:
https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.truthy

Reply:
No, it is about yamllint trying to be "smart", and assume truthy key names. It fails currently on this line: https://github.com/confluentinc/cp-ansible/blob/e7523d5011dc40a62c62ab3f1dcb78132f90c0a4/.github/workflows/linter.yml#L3, and that is why you had to ignore the whole file when adding this file.

Reply:
Oh so `on` is an invalid key... ok sure this is fine, and why do you think this needs to get into 6.1.1-post as opposed to waiting in the 6.1.x branch?

Reply:
Just because the file was added to that branch. But I can change the target branch to 6.1.x - no problem! 😀

Reply:
Changed target branch to 6.1.x.

Reply:
yes i regret adding that file to the 611post branch haha, but its not a showstopper, so i think its fine to fix in 61x

Reply:
yes i regret adding that file to the 611post branch haha, but its not a showstopper, so i think its fine to fix in 61x

Reply:
Ya haha I regret adding that file to 6.1.1-post, but this isnt really a showstopper so 61x should be fine